Added tooltip displaying the sync/charge status to the trayIcon

benedetto opened the pull request:
I've added a tooltip to the tray icon that displays the percentage of sync or charge completion while syncing/charging. Hence it is not necessary to switch to the openambit window to see how far the sync or charge process are done. Instead it's sufficient to hover with the mouse over the tray icon.

I've also added the respective translations as far as I was able to do so.

paddy-hack commented:
Cool.  Looks mostly fine to merge but what are the translation file changes about that introduce a `type="obsolete"` attribute?  If these are not related to your changes, I think they should not be part of the commit.
I'll forgive you the whitespace change in mainwindow.cpp :wink:

benedetto commented:
Well, looks like something in the tool chain went a bit havoc. I'm still new to C++/Qt, so I have to admit I just haven't seen those "obsolete" parts. I have removed them and also the whitespace change in mainwindow.cpp and updated my pull request.

paddy-hack commented:
In general, it's a good idea to only add the changes you want and review your changes before you commit :wink: 
```
git add --patch
git diff --cached
```
are your friends.

Looks good now.
